website/public/index.ejs.html

```diff
@@ -245,6 +246,8 @@ <h2 id="linter-feature-comparison">Features</h2>
               <td class="meh">open source<br />MIT</td>
```

Comment:
Blocking: Add a cell for SonarLint for the speed column. Write "?" if you're unsure about the speed; we can fill it in later after we run benchmarks.